client/src/main/java/org/apache/rocketmq/client/log/ClientLogger.java

```diff
@@ -35,16 +36,19 @@
 
     private static Logger createLogger(final String loggerName) {
         String logConfigFilePath =
-            System.getProperty("rocketmq.client.log.configFile",
-                System.getenv("ROCKETMQ_CLIENT_LOG_CONFIGFILE"));
+                System.getProperty("rocketmq.client.log.configFile",
+                        System.getenv("ROCKETMQ_CLIENT_LOG_CONFIGFILE"));
```

Comment:
Did you use the right code style? http://rocketmq.apache.org/docs/code-guidelines/